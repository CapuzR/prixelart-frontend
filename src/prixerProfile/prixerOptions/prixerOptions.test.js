import React from 'react';
import renderer from 'react-test-renderer';
import PrixerOptions from "./prixerOptions";

it('Login renders correctly', () => {
  const tree = renderer
    .create(<PrixerOptions />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});