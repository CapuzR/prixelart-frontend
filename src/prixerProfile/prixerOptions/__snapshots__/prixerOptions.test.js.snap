// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="MuiPaper-root makeStyles-paper-2 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      aria-label="text primary button group"
      className="MuiButtonGroup-root"
      role="group"
    >
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedText MuiButtonGroup-groupedTextHorizontal MuiButtonGroup-groupedTextPrimary MuiButton-textPrimary"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Artes
        </span>
      </button>
      <button
        className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButtonGroup-grouped MuiButtonGroup-groupedHorizontal MuiButtonGroup-groupedText MuiButtonGroup-groupedTextHorizontal MuiButtonGroup-groupedTextPrimary MuiButton-textPrimary"
        disabled={false}
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          Servicios
        </span>
      </button>
    </div>
  </div>
</div>
`;
